tests/test-xhr-credentials.js

```diff
@@ -1,4 +1,11 @@
 import assert from 'tjs:assert';
+import path from "tjs:path";
+
+// Cookies jar must be pre-created, otherwise this test will failed in Github Action
+const TJS_HOME = tjs.env.TJS_HOME ?? path.join(tjs.homeDir, '.tjs');
+const cookieJarPath = path.join(TJS_HOME, 'cookies');
+(await tjs.makeDir(TJS_HOME, { recursive: true }));
+(await tjs.open(cookieJarPath, 'w')).close();
```

Comment:
This shouldn't be necessary, the code should be able to work without an existing .tjs directory. See my comment above.